scripts/rollup-plugin-standalone-internals.js

```diff
@@ -0,0 +1,40 @@
+import fs from "fs";
+
+const standaloneURL = new URL("../packages/babel-standalone/", import.meta.url);
+const inStandalone = path => new URL(path, standaloneURL).pathname;
+
+const { noopPlugins, unexposedNoopPlugins } = JSON.parse(
+  fs.readFileSync(new URL("./scripts/pluginConfig.json", standaloneURL), "utf8")
+);
+
+const pluginUtilsID = "@babel/helper-plugin-utils";
+
+const generatedPluginsPath = inStandalone("./src/generated/plugins.ts");
+const makeNoopPluginPath = inStandalone("./src/make-noop-plugin.ts");
+const pluginUtilsShimPath = inStandalone("./src/plugin-utils-shim.ts");
+
+const toCamel = str => str.replace(/-[a-z]/g, c => c[1].toUpperCase());
```

Comment:
```suggestion
const toCamel = str => str.replaceAll(/-[a-z]/g, c => c[1].toUpperCase());
```
nit. And should this file be an `.mjs` file?

Reply:
There is a `package.json` with `"type": "module"`